root/user/components/UserInlineList.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/license/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import {l} from '../../static/scripts/common/i18n';
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import commaOnlyList from '../../static/scripts/common/i18n/commaOnlyList';
+
+type Props = {|
+  +editors: EditorT,
+|};
+
+const UserInlineList = ({editors}: Props) => {
+  <p>
+    {editors.size ? (
+      {commaOnList(
```

Comment:
Opening bracket is not appropriate here either, as you appropriately opened a bracket for JavaScript expressions as children on line 22 already. Just remove it (from line 23). (And remove closing bracket on line 28 accordingly.)